web-animations-1/Overview.bs

```diff
@@ -3960,9 +4111,164 @@ interface Animation : EventTarget {
+          and abort these steps.
+
+          <div class=issue>
+
+          The definition of [=being rendered=] [[!HTML]] with regards to
+          ''display: contents'' is still <a
+          href=https://github.com/whatwg/html/issues/1837>under discussion</a>.
+          For the purpose of this procedure, we assume that an
+          element with ''display: contents'' that otherwise would have
+          associated layout boxes (i.e. it is [=connected=] and not part of
+          a ''display: none'' subtree) <em>is</em> being rendered.
+
+          </div>
+
+     1.   Let |physicalProperties| be a [=ordered set|set=] of CSS property
+          names with the same contents as |longhandProperties| but
+          with each logical property name in |longhandProperties|
+          replaced with the [=equivalent physical properties=] based on the
+          computed value of 'writing-mode', 'direction', and
+          'text-orientation' for |target|.
+
+     1.   Let |inline style| be the result of getting the
+          [=CSS declaration block=] corresponding to |target|'s
+          [=style attribute=].
+          If |target| does not [=has an attribute|have=]
+          a [=style attribute=],
+          let |inline style| be a new empty [=CSS declaration block=] with
+          the <a for=CSSStyleDeclaration>readonly flag</a> unset and
+          <a for=CSSStyleDeclaration>owner node</a> set to |target|.
+
+     1.   Let |targeted properties| be the [=ordered set|set=]
+          of physical longhand properties
+          that are a [=target property=] for at least one [=animation effect=]
+          [=associated with an animation|associated=] with |animation|
+          whose [=target element=] is |target|.
+
+     1.   For each property, |property|, in |targeted properties|:
+
+          1.   If |filterProperties| is true and |physicalProperties|
+               does <em>not</em> include |property|, proceed to the next
+               property.
+
+          1.   Let |partialEffectStack| be a copy of the [=effect stack=]
+               for |property| on |target|.
+
+          1.   If |animation|'s [=replace state=] is
+               [=removed replace state|removed=],
+               add all [=animation effects=]
+               [=associated with an animation|associated=] with |animation|
+               whose [=target element=] is |target| and which include
+               |property| as a [=target property=] to |partialEffectStack|.
+
+          1.   Remove from |partialEffectStack| any [=animation effects=]
+               whose [=associated with an animation|associated=] [=animation=]
+               has a higher [=composite order=] than |animation|.
+
+          1.   Let |effect value| be the result of calculating the result of
+               |partialEffectStack| for |property| using |target|'s computed
+               style (see [[#calculating-the-result-of-an-effect-stack]]).
```

Comment:
Can this still result in growing lists? I.e. if we have composite: 'add' transforms might this result in an ever growing transform list or is the effect value the transform matrix? What about filter lists?

Reply:
Yes, it can. The computed value for transforms is a list, it just happens to be [serialized as a single matrix value](https://drafts.csswg.org/css-transforms/#serialization-of-the-computed-value). For filters too, there is a similar issue.